### PR TITLE
GAME-18 - Styling Game.html file

### PR DESCRIPTION
**Pull request by Birek100**
Add menu and styling in game.html file, update style.css file containing changes in game.html.

### CODE REVIEW

File: public/styles.css [hunk 1]
```diff
@@ -1,29 +1,79 @@
...
 	margin-left: auto;
 	margin-right: auto;
+	overflow: auto;
 }
-footer {
-	margin-top: 15px;
+
+.masthead {
+	text-align: center;
+	font-family: 'Lobster', cursive;
```
Comment (oskarszura) [inline public/styles.css]: Do you have this font implemented into browers?

File: public/styles.css [hunk 2]
```diff
@@ -1,29 +1,79 @@
...
+	font-size: 38px;
 }
+
```
Comment (oskarszura) [inline public/styles.css]: Why two new lines?

File: public/styles.css [hunk 3]
```diff
@@ -1,29 +1,79 @@
...
+	width: 25%;
+	background: #7e7e77;
+	float: left;
+	padding: 20px;
+	min-height: 540px;
+	border: 2px dashed;
+	font-size: 18px;
+	text-align: center;
+	display:inline;
+
```
Comment (oskarszura) [inline public/styles.css]: Why blank space?

File: public/styles.css [hunk 4]
```diff
@@ -1,29 +1,79 @@
...
+.foot {
+	clear: both;
+	text-align: center;
+	padding: 20px;
+	font-size: 18px;
+	color: #4e2489;
+}
+
+
```
Comment (oskarszura) [inline public/styles.css]: Why blank space?

File: views/game.html [hunk 5]
```diff
@@ -1,11 +1,39 @@
...
 <body> 
-	<div>
-		The GAME
+	<div class="container">
+		<header>
+			<div class = "masthead">
+				The Penguin Game 
+			</div>
+		</header>
+
```
Comment (oskarszura) [inline views/game.html]: Why two blank spaces? HTML tags should be well formatted.

File: views/game.html [hunk 6]
```diff
@@ -1,11 +1,39 @@
...
+			<div class = "masthead">
+				The Penguin Game 
+			</div>
+		</header>
+		
+
+		<aside>
+			<div class="menu">
+				
+					Menu
```
Comment (oskarszura) [inline views/game.html]: ```<div class="menu">Menu</div>``` you don't need so many spaces.

File: views/game.html [hunk 7]
```diff
@@ -1,11 +1,39 @@
...
+				
+			</div>
+		</aside>
+
```
Comment (oskarszura) [inline views/game.html]: Blank space.

File: views/game.html [hunk 8]
```diff
@@ -1,11 +1,39 @@
...
+		<section>
+			<div class ="content">
+				Play The Game!!!
+			</div>
+
+		</section>
+
```
Comment (oskarszura) [inline views/game.html]: Two blank spaces.

File: views/login.html [hunk 9]
```diff
@@ -1,31 +1,49 @@
 <html>
+
```
Comment (oskarszura) [inline views/login.html]: Why do you add blank spaces?

File: views/login.html [hunk 10]
```diff
@@ -1,31 +1,49 @@
...
 <head>
 	<link rel="stylesheet" type="text/css" href="/static/styles.css">
 	<script src="/static/scripts.js"></script>
+	<link href="https://fonts.googleapis.com/css?family=Lobster&display=swap" rel="stylesheet">
 	<title>Penguin Game</title>
 </head>
+
```
Comment (oskarszura) [inline views/login.html]: Blank space.

**Comment by Birek100** [comment]
now it should be ok

File: public/styles.css [hunk 11]
```diff
@@ -1,29 +1,71 @@
+html {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+*, *:before, *:after {
+  -webkit-box-sizing: inherit;
```
Comment (oskarszura) [inline public/styles.css]: What 'inherit' does?

File: public/styles.css [hunk 12]
```diff
@@ -1,29 +1,71 @@
+html {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
```
Comment (oskarszura) [inline public/styles.css]: What 'box-sizing' does?

File: public/styles.css [hunk 13]
```diff
@@ -1,29 +1,71 @@
...
 }
 form input {
 	padding: 5px;
 	border-radius: 5px;
 }
 div.register_html {
 	text-align: center;
+	margin-top: 10px;
+}
+div.Login_html {
```
Comment (oskarszura) [inline public/styles.css]: I've told you, you should use BEM naming convetion. Fix it.

File: public/styles.css [hunk 14]
```diff
@@ -1,29 +1,71 @@
...
-	margin: auto;
-	margin-bottom: 10px;
+	text-align: center;
+	margin-top: 10px;
 }
 form input {
 	padding: 5px;
 	border-radius: 5px;
 }
 div.register_html {
```
Comment (oskarszura) [inline public/styles.css]: It's not BEM naming convetion.

File: public/styles.css [hunk 15]
```diff
@@ -1,29 +1,71 @@
...
+	text-align: center;
+}
+.foot {
+	clear: both;
+	text-align: center;
+	padding: 20px;
+	font-size: 18px;
+	color: #4e2489;
 }
 form{
```
Comment (oskarszura) [inline public/styles.css]: Lack space between form and {

File: public/styles.css [hunk 16]
```diff
@@ -1,29 +1,71 @@
...
 	padding: 15px;
 }
 form div.register {
```
Comment (oskarszura) [inline public/styles.css]: Is div. necessary here?

File: views/game.html [hunk 17]
```diff
@@ -1,11 +1,24 @@
...
-	<title>The GAME</title> 
+	<link href="https://fonts.googleapis.com/css?family=Lobster&display=swap" rel="stylesheet">
+	<script src="/static/scripts.js"></script>
+	<title>The Game</title> 
 </head> 
 <body> 
-	<div>
-		The GAME
+	<div class="container">
+		<header class = "masthead">
```
Comment (oskarszura) [inline views/game.html]: blank spaces between class and = and "masthead"

File: views/game.html [hunk 18]
```diff
@@ -1,11 +1,24 @@
...
+			The Penguin Game 
+		</header>
+		<div class="menu">
+			Menu
+		</div>
+		<div class ="content">
```
Comment (oskarszura) [inline views/game.html]: Unnecessary blank spaces

File: views/game.html [hunk 19]
```diff
@@ -1,11 +1,24 @@
...
+			Play The Game!!!
+		</div>
+		<footer class = "foot">
```
Comment (oskarszura) [inline views/game.html]: Unnecessary blank spaces.

File: views/login.html [hunk 20]
```diff
@@ -1,39 +1,33 @@
...
 	<script src="/static/scripts.js"></script>
 	<title>Penguin Game</title>
 </head>
-
 <body>
 	<div class="container">
-		
-		<header>
-			<h1>Penguin Game</h1>
+		<header class = "masthead">
```
Comment (oskarszura) [inline views/login.html]: Unnecessary blank spaces

File: views/login.html [hunk 21]
```diff
@@ -1,39 +1,33 @@
...
-			<input type="submit" value="Play">
-		</div>
-	</form>
-	</section>
-	
-	<footer>
-		All rights reserved &copy
-	</footer>
-	
+			<div class="Login_html">
```
Comment (oskarszura) [inline views/login.html]: Bad indentation

File: views/login.html [hunk 22]
```diff
@@ -1,39 +1,33 @@
...
+					<div class="register">
+						<label for="password">Password</label><br>
+						<input name="password" type="password" required>
+					</div>
+					<div class="register">
+						<input type="submit" value="Play">
+					</div>
+				</form>
+			</div>
+		<footer class = "foot">
```
Comment (oskarszura) [inline views/login.html]: Unnecessary blank spaces

File: views/register.html [hunk 23]
```diff
@@ -1,26 +1,33 @@
...
-				<input name="password" type="password" require>
-			</div>
-			<div class="register">
-				<input type="submit" value="Submit">
-			</div>
-		</form>
-	</div> 
+<body> 	
+	<div class="container">
+		<header class = "masthead">
```
Comment (oskarszura) [inline views/register.html]: Unnecessary blank spaces

File: views/register.html [hunk 24]
```diff
@@ -1,26 +1,33 @@
...
+			Penguin Game
+		</header>
+			<div class="Register_html">
```
Comment (oskarszura) [inline views/register.html]: Bad indentation

File: views/register.html [hunk 25]
```diff
@@ -1,26 +1,33 @@
...
+					<div class="register">
+						<label for="password">Password</label><br>
+						<input name="password" type="password" require>
+					</div>
+					<div class="register">
+						<input type="submit" value="Submit">
+					</div>
+				</form>
+			</div> 
+		<footer class = "foot">
```
Comment (oskarszura) [inline views/register.html]: Unncessary blank spaces

**Comment by Birek100** [comment]
Now i think it's ok. Please look at it

File: views/register.html [hunk 26]
```diff
@@ -1,26 +1,31 @@
...
-				<input type="submit" value="Submit">
-			</div>
-		</form>
-	</div> 
+<body> 	
+	<div class="container">
+		<header class ="masthead">
+			Penguin Game
+		</header>	
+			<form class="login-form" action="/register" method="post" autocomplete="off">
```
Comment (oskarszura) [inline views/register.html]: Indentation is bad

File: views/login.html [hunk 27]
```diff
@@ -1,39 +1,31 @@
...
-	</section>
-	
-	<footer>
-		All rights reserved &copy
-	</footer>
-	
+		<header class="masthead">
+			Penguin Game
+		</header>	
+			<form class="login-form" action="/login-auth" method="get" autocomplete="off">
```
Comment (oskarszura) [inline views/login.html]: Indentation is bad

File: views/register.html (hunk 26 above)
Comment (Birek100) [reply views/register.html]: between 12 and 13 line ??

File: public/styles.css (hunk 12 above)
Comment (Birek100) [reply public/styles.css]: box sizing is propetory that tells how height and witdth is calculated

File: views/register.html (hunk 26 above)
Comment (oskarszura) [reply views/register.html]: `form` has too deep indentaiton - it should be aligned with `header` and `footer`

**Comment by oskarszura** [comment]
Merge it :shipit: !

**Pull request merged**